pipeline/load.py

```diff
@@ -21,14 +22,100 @@ def create_database_connection(config: _Environ) -> Connection:
     return conn
 
 
+def insert_into_location_table(connection: Connection, plant_data: pd.DataFrame) -> None:
+    """Inserts data from a pandas data frame into a SQL Server into the location table."""
+
+    plant_list = plant_data.values.tolist()
+
+    for plant in plant_list:
+
+        query = sql.text(
+            "SELECT location_id FROM s_delta.location WHERE region = (:region)")
+        args = ({"region": plant[9]})
+        id = connection.execute(query, args).fetchone()
+
+        if id is None:
+            region = plant[9]
+            continent = plant[10]
+            country = plant[11]
+
+            query = sql.text(
+                """INSERT INTO s_delta.location (region,country,continent)
+                VALUES (:region,:continent,:country)""")
+            connection.execute(
+                query, {"region": region, "continent": continent, "country": country})
+
+
+def insert_into_botanist_table(connection: Connection, plant_data: pd.DataFrame) -> None:
+    """Inserts data from a pandas data frame into a SQL Server into the location table"""
+
+    plant_list = plant_data.values.tolist()
+
+    for plant in plant_list:
+
+        connection.execute(sql.text("USE plants;"))
+
+        query = sql.text(
+            "SELECT botanist_id FROM s_delta.botanist WHERE telephone_number = (:telephone)")
+        args = ({"telephone": plant[8]})
```

Comment:
Just a reallyyy nitpicky thing, in the SQL schema, I think Zander set the email address as the unique constraint, but here we're using the phone number. I know they're both unique, but maybe worth just going with one for standardisation :)